src/InvalidArgumentException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Kodus\Cache;
+
+class InvalidArgumentException extends \InvalidArgumentException implements \Psr\SimpleCache\InvalidArgumentException
+{
```

Comment:
It's the same name, if we did that, we should make alias's.... And it's just an implementation detail because simple-cache has decided on using it's own `InvalidArgumentException`, and not the generic one provided by PHP